Bottom: 1312348f2d7bf9de0233857fb50a703749c4d3e2
Top:    1d339fecf151b649360c944ed23fa48e9983cd9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 23:27:01 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 25ea1c3..716b850 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1902,8 +1902,8 @@ ScrollAction (int argc, char **argv, int x, int y)
   else
     AFAIL (scroll);
 
-  notify_crosshair_change (false);
   ghid_port_ranges_pan (dx, dy, TRUE);
+  notify_crosshair_change (false);
   MoveCrosshairRelative (dx, dy);
   AdjustAttachedObjects ();
   notify_crosshair_change (true);
